services/dkg/pedersen/mod.go

```diff
@@ -95,20 +102,29 @@ func (s *Pedersen) Listen(electionIDBuf []byte) (dkg.Actor, error) {
 		return actor, xerrors.Errorf("actor already exists for electionID %s", electionID)
 	}
 
-	return s.NewActor(electionIDBuf, NewHandlerData())
+	err := txmngr.Sync()
```

Comment:
Are you sure we need to sync there ?

Reply:
Not anymore you're right, it was for the scenario test but in the end I just synced earlier in the handler instead